keyboards/sergiopoverony/creator_pro/readme.md

```diff
@@ -0,0 +1,28 @@
+# Creator Pro
+
+![Creator Pro Layout](https://i.imgur.com/0PmpLcM.png)
+
+This Keypad has 1 rows x 9 columns of keys. It has the top/default layer that
+has a few 1-8 keys as well as one dedicated encoder key for change layer. The
+ second layer has QWERASDF keys. Encoder have 4 function on any layer is 
+ Volume Control, Mouse Wheel, PageUp/PageDn, Left/Righ buttons.
+The keypad also includes three RGB LEDs that show active layer.
+
+Keyboard Maintainer: [SergioPoverony](https://github.com/sergiopoverony)  
+
+Hardware:
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware?

Reply:
Add in Hardware section